docs/Documentation/Systems/Vermilion/modules.md

```diff
@@ -85,5 +86,5 @@ The python in this environment is very up to date, version 3.10.0.  It also cont
 
 ## Examples
 
-The directory /nopt/nrel/apps/210929a/example contains some simple build and run scripts.  The directory /nopt/nrel/apps/210929a/example/vasp contains information about running Vasp.  These are discussed in more detail in the page [Running](./running.md).
+The directory /nopt/nrel/apps/210929a/example contains some simple build and run scripts.  The directory /nopt/nrel/apps/210929a/example/vasp contains information about running VASP.  These are discussed in more detail in the page [Running on Vermilion](./running.md).
```

Comment:
I don't think we should be pointing users to a directory. If any information is needed from this directory, we can add it to the VASP on Vermillion section. I'll add a note about this to the issues I just opened.

Up to you on how to handle the general example directory

Reply:
Do you think the scripts in the example directory should be copied to the code repo? (I'll take a look at them myself once systems are returned to service)

Is all important info captured from the vasp directory? If so, I'll remove that reference.

Reply:
Vermillion is currently hanging on log in so I can't really go check but last I did I didn't think there was much useful there that we didn't have in the documentation, which John has pointed out is out of date. I think trying to keep 2 places up to date is more difficult than one... Maybe I can add a note to check this directories when John and I work through VASP on Vermillion?

Reply:
I agree; removed these paths from the page and added issue #525 to confirm content in them is captured.